ci/docker/runtime_functions.sh

```diff
@@ -1524,12 +1524,15 @@ nightly_scala_demo_test_cpu() {
     bash bin/run_im.sh
 }
 
-nightly_estimator() {
+nightly_estimator_numpy_global_flag() {
```

Comment:
Sorry I'm not following. Why is this even a nightly test?

Also, I don't think that this will work. When you change the name of the function but not the point where it gets referenced (IF it gets referenced), then it would break, right? But lets forget about that.

Why don't you make this a normal unit test? The test doesn't seem to fit into a constraint that warrants nightly execution.

Reply:
The feature is only going to be used in MXNet subproject (even numpy willl not use), so I think it make sense to put in nightly test.

I changed the tests/nightly/JenkinsfileForBinaries to reference this function. Do I still miss something?

The test should only be run alone, since it affect the behavior of numpy shape in other tests that is run parallelly. i.e. It will crash our operator unit test if I put this in noraml unit test

Reply:
Could you elaborate which subproject?

Oh right, you change the Jenkinsfile. Sorry about that!

We don't parallelize our tests, so it should be fine to just run it together with the others. The only thing you have to watch is to clean up properly.

Reply:
To me this seems like a normal feature - independent of where it's used, so I'd be in favour if we add a regular unit test.

Reply:
The project is not open source yet.

AFAIK, nosetest run tests in parallel. Currently numpy shape is claimed as thread_local variable, this feature enable global functionaility where we can see the flag from any thread. It means we might break non-numpy-compatible operator test that is run parallelly because MXNet engine treat them as numpy-compatible.

Although the payload is small, I don't want MXNet developers bothered by this test and it's actually more like an integration test.

Reply:
We run nosetests without parallelization :)

Reply:
ok I'll try to add this in regular unit test